pkgs/applications/editors/emacs/generic.nix

```diff
@@ -15,7 +15,7 @@
 , libgccjit, targetPlatform, makeWrapper # native-comp params
 , fetchFromSavannah
 , systemd ? null
-, withX ? !stdenv.isDarwin
+, x11Support ? !stdenv.isDarwin
```

Comment:
This makes the x11 option inconsistent with all other options in the derivation.
I don't really like it.

Reply:
I can rename them all. E.g `withMotif` -> `motifSupport`. You okay with that?

Reply:
@adisbladis ping?